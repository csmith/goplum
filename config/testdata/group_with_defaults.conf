# Test that group blocks can contain defaults blocks
group "production" {
  alert_limit = 5
  alert_window = 10m
  
  defaults {
    interval = 30s
    timeout = 20s
    alerts = ["pagerduty", "email"]
  }
}