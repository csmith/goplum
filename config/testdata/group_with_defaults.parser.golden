{
  "DefaultSettings": {},
  "AlertBlocks": null,
  "CheckBlocks": null,
  "PluginSettings": null,
  "GroupBlocks": [
    {
      "Name": "production",
      "Type": "",
      "Settings": {
        "alert_limit": 5,
        "alert_window": 600000000000,
        "defaults": {
          "alerts": [
            "pagerduty",
            "email"
          ],
          "interval": 30000000000,
          "timeout": 20000000000
        }
      }
    }
  ]
}